### DISCUSSION TITLE
git clone시 특문

### DISCUSSION THREAD

**Discussion by squirrelabbit** [yoonjeong-kwon/fastcampus-kubernetes#27] [Q&A] [answered]
안녕하세요 강의 잘 듣고 있습니다.
다만 이미지 파일인 images/?check-gcp-freetier.png에서의 특문인 ? 때문에
윈도우에서는 깃 클론이 되지 않고 있습니다. 
찾아본 결과로는 파일이름을 바꿔야만 하는데 
잘 해결되지 않아 질문으로 남깁니다.

감사합니다.

**Comment by yoonjeong-kwon** [comment, marked as answer]
@squirrelabbit 안녕하세요! 제가 개인사정으로 한국에 그동안 없어서 답변이 늦어졌습니다. 죄송합니다. 이미지는 확인해서 수정하고 한 번 더 코멘트로 말씀드리겠습니다.
확인해주셔서 감사합니다.

**Reply by yoonjeong-kwon** [reply]
@squirrelabbit 말씀주신대로 파일명 앞에 특수문자가 섞여 있었네요. 
확인해서 파일명 변경해두었습니다. 
또 문제가 발생하면 말씀해주세요. 감사합니다! 😃
